### ISSUE TITLE
With voice over, unable to access message, republish, favorite, share and more buttons

### ISSUE THREAD

**Issue by ldzintegratore** [metabolist/metatext#229]
Coming soon. It's too long to describe now!
But it is impossible to navigate in the area

iOS Device

**Comment by alexbbrown** [comment]
I checked on my device and verified that Navigating using voiceover cursor to reply, boost and favourite is not possible in the message list views including the feeds.

But it works fine in the detail views.  

In list view you can still reply and boost and favourite using the thoughtful support for accessibility actions.

Workaround 1
in a list of messages (eg the local feed)
Navigate to a message you want to boost
double tap on current message in list view to open it in detail view 
then navigate to the boost (or reply, favorite) button

![image](https://user-images.githubusercontent.com/2532344/203488257-4121ce21-7796-4419-96fc-e0c0d2ca5a12.jpeg)

Workaround 2
In a list of messages such as the local feed
Two finger twist to turn the voiceover rotor to Actions
Swipe up and down until you find the action you want, e.g. Boost
Double tap to activate the boost action

![image](https://user-images.githubusercontent.com/2532344/203488057-8e29d8be-d546-4cfb-a2ce-b7d67343daa6.jpeg)



Update: changed "triple tap" to "double tap" in workaround 1.

**Comment by alexbbrown** [comment]
The current UI design seems consistent with the guidance given in this video : https://developer.apple.com/wwdc18/230

**Comment by ldzintegratore** [comment]
Yes, I just looked at it and it actually works well like this. Great!
How does the user know to triple tap to access the detail view? There is no indication that the component is a link or a button to access the detail view?

**Comment by alexbbrown** [comment]
@ldzintegratore in my experiments, both the default action (double tap) and alternative action (triple tap) for a toot in a toot list is to show the detail view.  I agree that it doesn't mention that. 

Perhaps an action hint is appropriate here. Im not an expert.

For me fact that it says actions available is enough to prompt me to explore those actions.

**Comment by ldzintegratore** [comment]
It seems that on other applications the behavior is about the same.

Some applications propose to make available the number of replies or bookmarks...

The actions of reply, bookmark... are available by sliding down or up

Is this an obvious practice, I don't know

Thank you for your answers and your interest in making the application accessible.

**Comment by alexbbrown** [comment]
You can turn on reply and favorite counts in the meta text app settings.

**Comment by alexbbrown** [comment]
@ldzintegratore can this issue be closed? If not can you describe what specific app behaviors still are causing a problem?

**Comment by ldzintegratore** [comment]
Yep, you can close the ticket
Thx
